packages/webapp/pages/squads/[handle].tsx

```diff
@@ -87,32 +96,12 @@ export async function getStaticPaths(): Promise<GetStaticPathsResult> {
 interface SquadPageParams extends ParsedUrlQuery {
   handle: string;
 }
-
-export async function getStaticProps({
+export function getStaticProps({
   params,
-}: GetStaticPropsContext<SquadPageParams>): Promise<
-  GetStaticPropsResult<SourcePageProps>
-> {
-  try {
-    const squad = await getSquad(params.handle);
-    const squadMembers = await getSquadMembers(squad.id);
-    return {
-      props: {
-        squad,
-        squadMembers,
-      },
-      revalidate: 60,
-    };
-  } catch (err) {
-    if (err?.response?.errors?.[0].extensions.code === 'NOT_FOUND') {
-      return {
-        props: {
-          squad: null,
-          squadMembers: [],
-        },
-        revalidate: 60,
-      };
-    }
-    throw err;
-  }
+}: GetStaticPropsContext<SquadPageParams>): GetStaticPropsResult<SourcePageProps> {
+  return {
+    props: {
+      handle: params.handle,
+    },
+  };
```

Comment:
Was this change intentional? I was having issues here too, we can probably fix this function at a later stage though we need to create a ticket I think. So we won't forget.

Reply:
Yeah we moved away from SSR due to needing the auth for squads + additional user related information.
But still wanted to leverage the handle passing instead of getting it from the router.

Hope that makes sense?

Reply:
Oh, that sounds amazing. Thanks for letting me know 💯